### PR TITLE
🔧 Consultancy page about section (first) #179

### PR DESCRIPTION
**Pull request by kiplangat01**
# Description

implemented Consultancy Page About Section (first) #179 . This component reuses the banner component created in #8. 

##screenshot
Screenshot from 2023-09-09 11-55-05

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/pages/consultancy-page/src/lib/consultancy-page.module.ts [hunk 1]
```diff
@@ -4,9 +4,13 @@ import { CommonModule } from '@angular/common';
 import { ConsultancyRoutingModule } from './consultancy.routing';
 
 import { ConsultancyPageComponent } from './main/consultancy-page/consultancy-page.component';
+import { ElewaConsultancyAboutOneComponent } from './components/elewa-consultancy-about-one/elewa-consultancy-about-one.component';
+import { BannersModule } from '@elewa-website/elements/banners';
+import { MainPageModule } from '@elewa-website/elements/layout/pages/main-page';
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/consultancy-page.module.ts:10]: Kindly, observe the order of imports; elements imports should appear before local imports

File: libs/features/pages/home/src/lib/features-pages-home.module.ts [hunk 2]
```diff
@@ -8,9 +8,16 @@ import { HomePageComponent } from './main/home/home-page.component';
 import { NewsSectionComponent } from './components/news-section/news-section.component';
 
 import { HomeRoutingModule } from './home.routing';
+import { ConsultancyPageModule } from '@elewa-website/features/pages/consultancy-page';
```
Comment (AnnaL001) [inline libs/features/pages/home/src/lib/features-pages-home.module.ts:12]: Same here, the order of imports; features imports should appear before local imports

**Review by AnnaL001 — CHANGES_REQUESTED** [review]
Good job so far, just a couple of adjustments

File: libs/features/pages/home/src/lib/features-pages-home.module.ts (hunk 2 above)
Comment (kiplangat01) [reply libs/features/pages/home/src/lib/features-pages-home.module.ts]: corrected

**Comment by LemmyMwaura** [comment]
Sync the latest changes. I've updated the pill shape on the image container

**Comment by kiplangat01** [comment]
> Sync the latest changes. I've updated the pill shape on the image container

it change but it is still on Horizontal position

**Comment by LemmyMwaura** [comment]
Good work but this has been resolved

**Pull request closed without merging**